common/src/main/java/com/sequenceiq/cloudbreak/telemetry/streaming/StreamingConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.sequenceiq.cloudbreak.telemetry.streaming;
+
+public interface StreamingConfiguration {
```

Comment:
here you could use some constant defaults for worker and size limit that makes more sense (even if they are overridden) like worker: 1, size limit: 200